src/artemis/snapshot_plan.py

```diff
@@ -1,13 +1,21 @@
 from bluesky import RunEngine
 from bluesky.plan_stubs import mv, abs_set, wait
 from devices.backlight import Backlight
+from devices.aperture import Aperture
 
-backlight = Backlight(name="Backlight", prefix=f"BL03I")
+backlight = Backlight(name="Backlight", prefix="BL03I")
+aperture = Aperture(name="Aperture", prefix="BL03I")
+aperture.wait_for_connection()
 
 RE = RunEngine({})
 
-def move_then_wait():
-	yield from abs_set(backlight.pos, Backlight.OUT, group='A')
+def prepare_for_snapshot():
+	yield from abs_set(backlight.pos, Backlight.IN, group='A')
+	# TODO get from beamlineParameters
+	miniap_y_ROBOT_LOAD = 31.40
```

Comment:
**nit**: I realise this is a bit temporary until we figure out how to do beamlineParameters but can we rename this to be more snake_case/descriptive. It's also not clear why we're using this position given that the robot isn't being used here at all. So maybe either rename to something like `aperture_y_out_of_beam` or a comment as to why we're referencing the robot.

Reply:
I've added a comment to https://github.com/DiamondLightSource/python-artemis/issues/53 that we need to get `beamlineParams` working ASAP

Reply:
The name was pulled straight from beamlineParameters, but have updated it in https://github.com/DiamondLightSource/python-artemis/pull/121/commits/d3c387f99b968f6e50fc7b6926d5d76866a09247